Poker.BE/Poker.BE.Domain/Game/Room.cs

```diff
@@ -139,13 +139,13 @@ public void EndCurrentHand()
         /// UC027 Choose Play Move
         /// </summary>
         /// <see cref="https://docs.google.com/document/d/1OTee6BGDWK2usL53jdoeBOI-1Jh8wyNejbQ0ZroUhcA/edit#heading=h.8f3okxza6g2d"/>
-        public void ChoosePlayMove(Round.Move move)
+        public void ChoosePlayMove(Round.Move move, int amountToBet)
```

Comment:
❓  - after the player had chose his move, how do he know what's the next sets of moves he can do? i thought this method will return an `ICollection<Round.Move>` so the user will know what he can do next.. depending what he already did, or what the other players at the table had done... right?